### PR TITLE
Fix #659 - Show Lottie Animation when there are no Favourite Items

### LINKED ISSUE #659: [Favourites] Show Lottie Animation when there are no favourite items
### Changes to be done
* Change FAB `+` icon colour to white.
* When there aren't any favourite items show following Lottie animation.
      https://lottiefiles.com/347-like
* Change animation's colour from Red to App's Theme using Lottie Editor

### PR DESCRIPTION
**Pull request by piedcipher**
# Description

- [This](https://lottiefiles.com/347-like) animation is shown when there are no favourites items. (Colour of this animation is changed according to app theme colours using Lottie Editors)

**Demo**

Fixes #659

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by vyankatesh24** [comment]
@piedcipher 
Increase the size of animation view, check if looks good with 120x120 dp

File: app/src/main/java/swati4star/createpdf/fragment/FavouritesFragment.java [hunk 1]
```diff
@@ -54,6 +56,7 @@
         implements SharedPreferences.OnSharedPreferenceChangeListener,
         View.OnClickListener {
     private SharedPreferences mSharedpreferences;
+    private boolean mDoesFavouritesExist = false;
```
Comment (vyankatesh24) [inline app/src/main/java/swati4star/createpdf/fragment/FavouritesFragment.java]: this assignment of `false` has no use right? As you are assigning again with same at line 155.

Comment (piedcipher) [reply app/src/main/java/swati4star/createpdf/fragment/FavouritesFragment.java]: Yes, this initialisation can be removed.

**Comment by piedcipher** [comment]
Adding quick Screenshot without Code after changing size to 120dp.

**Comment by vyankatesh24** [comment]
> Adding quick Screenshot without Code after changing size to 120dp.
> 
> 

Ok Keep 120 dp

File: app/src/main/java/swati4star/createpdf/fragment/FavouritesFragment.java (hunk 1 above)
Comment (piedcipher) [reply app/src/main/java/swati4star/createpdf/fragment/FavouritesFragment.java]: Solved.

**Pull request merged**